src/libraries/System.Private.CoreLib/src/System/Runtime/Intrinsics/Vector128.cs

```diff
@@ -567,6 +575,7 @@ static Vector128<float> SoftwareFallback(float value)
         /// <returns>A new <see cref="Vector128{UInt16}" /> with all elements initialized to <paramref name="value" />.</returns>
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
         [CLSCompliant(false)]
+        [Intrinsic]
```

Comment:
I am wondering if we should take this PR now and after #35037 is approved re-visit this and re-implement the functionality of Vector64/128.Create() with one operand in C# using AdvSimd.DuplicateToVector64/128(). 

Basically, this would mean that we leverage the JIT implementation of Vector64/128.Create() for DuplicateToVector64/128() (this would require only cosmetic changes in the JIT) and add the following to the C# files
```c#
if (AdvSimd.IsSupported)
{
  return AdvSimd.DuplicateToVector128(value);
}
```

This would be keep consistency with implementation on x86
@tannergooding @kunalspathak What do you think?

Reply:
I didn't know about #35037. I don't see any problem doing what you are suggesting given there are no much perf implication (and I don't think there will be any).

Reply:
I think it might be better to make keep this intrinsic and have `DuplicateToVector128` share a codepath with it or forward to it as a helper.

The reasoning is that we will eventually want to support creating constants for all the various `Create` methods on both x86, x64, and ARM64 (and presumably ARM32 if that is ever supported).
The mapping of `Create` to underling intrinsics isn't necessarily 1-to-1 (especially on x86/x64 where containment and different paths exist depending on what ISAs are available).

So, as we add more paths to the JIT that use these intrinsics and even potentially look at introducing other optimizations that use the HWIntrinsics, it would be convenient to be able to just remember that `Vector128_Create` is what you use on either architecture for efficiently creating a vector.

Reply:
@tannergooding Agree